<?xml version="1.0" encoding="UTF-8"?>
<launch>
  
  <arg name="show" default="true" />
  <node pkg="amonitor_camera" type="amonitor_camera_node" name="marker_map" output="screen">
    <param name="show_output_video" value="$(arg show)" />

    <param name="markermap_config" value="$(find amonitor_camera)/marker_map/map_home/map.yml" />
    <param name="marker_size" value="0.15" />
    
    <param name="publish_to_rviz" value="true" />
    <param name="debug_save_input_frames" value="false" />
    <param name="debug_save_output_frames" value="false" />
    
    <remap from="input_image" to="/videofile/image_raw" />
    <remap from="output_image" to="/marker_map/position" />
  </node>
</launch>